### ISSUE TITLE
Resize img changes the img tag for canvas tag

### ISSUE THREAD

**Issue by ruben-murcia** [Alex-D/Trumbowyg#1158]
When inserting an image by link or by uploading a file, we get in the HTML an img tag

```html
<img src="https://SOME_URL/PIC.png" alt="" width="159" height="66">
```

And when the user resizes the image with his mouse (having the resizeimg plugin loaded), the img tag changes to a canvas tag

```html
<canvas tabindex="0" id="trumbowyg-resizimg-1598581276115" width="169" height="72" class="resizable" style="margin: -5px; touch-action: none; cursor: default;"></canvas>
```

Once the user clicks on the Editor area but outside the image, it becomes an img tag again.

The problem here is when the user submits the content when his last change was resizing the img and it is a canvas tag.
We lose the img tag with the img url, and we just have a canvas tag that does not display anything.

To solve that, I think the easiest way is to just move the caret to the beginning of the HTML, forcing the canvas tag to become an img tag, but even using `saveRange` and `restoreRange` when clicking on the submit button, it does not work.

Is there an easy way to move the caret at the beginning? Any other idea of how to fix it?

Thanks!

**Comment by lazybird** [comment]
Hi @ruben-murcia,
Thanks for your description of this bug - It helped me understand, I was a bit lost with this problem.

Any success on resolving this issue ? It's quite annoying, right ?

**Comment by Alex-D** [comment]
Should be fixed in v2.22 (will be released soon)
